composer.json

```diff
@@ -11,20 +10,25 @@
     }
   ],
   "require": {
-    "php": "^7.4",
-    "roave/you-are-using-it-wrong": "^1.3"
+    "php": ">= 7.4",
+    "roave/you-are-using-it-wrong": "^1.4"
+  },
+  "require-dev": {
+    "drupol/php-conventions": "^1.7.1",
+    "friends-of-phpspec/phpspec-code-coverage": "^4.3.2",
```

Comment:
Probably best to skip code coverage altogether, and jump directly towards mutation test coverage

Reply:
It would require to rewrite all the tests with PHPSpec or PHPunit then ?

Reply:
Hmm, is there no phpspec backend for infection? 🤔

Reply:
Current tests are written with Kahlan, AFAIK, there is no adapter for Kahlan and Infection.

This is why I suggested to rewrite using PHPSpec or PHPunit, they both works with Infection.